src/nublado/controller/services/builder/lab.py

```diff
@@ -570,14 +597,17 @@ def _build_pod(
             self._build_pod_secret_volume(user.username),
             self._build_pod_env_volume(user.username),
             self._build_pod_tmp_volume(mem_size=resources.limits.memory),
+            self._build_pod_startup_volume(mem_size=640 * 1024),
```

Comment:
Is there any chance we'd want this size configurable? Or perhaps moved into a constant?

Reply:
The comment wouldn't fit on the line, but it's "640k ought to be enough for anyone."

That's grossly too much: it's two JSON documents; there are maybe fifteen arguments in one and maybe 60 environment variables in the other; each one fits on an 80-column line, so 6K would be enough.  Eventually we're gonna trade environment for args, but the net amount should remain the same.

But also, what's 640k in this day and age?

Reply:
Comment added, even though it spoils the joke.